reframe/core/systems.py

```diff
@@ -365,13 +365,11 @@ def devices(self):
 
     @property
     def extras(self):
-        '''User defined attributes of the system.
-
-        By default, it is an empty dictionary.
+        '''User defined properties defined in the configuration.
 
         .. versionadded:: 3.5.0
 
-        :type: :class:`object`
+        :type: :class:`Dict[str, object]`
```

Comment:
Is this correct? Now in the schema of the configuration we only check that is a valid object, right? Same for the `extras` of the environments.

Reply:
Yes, object properties in JSON are strings, whereas their values can be anything. And JSON objects are translated to Python dicts.